src/main/webapp/app/entities/fda-submission/fda-submission.tsx

```diff
@@ -77,6 +79,63 @@ export const FdaSubmission = (props: IFdaSubmissionProps) => {
 
   const { match } = props;
 
+  const columns: Column<IFdaSubmission>[] = [
+    {
+      id: 'id',
+      accessor: 'id',
```

Comment:
I remember if the accessor is not specified, it uses id. Could you take a look? Though I do like list accessor always to make it clear.

Reply:
If the accessor is a string (not function), then `id` is not necessary. I will remove the `id` field.